mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -26,7 +27,7 @@
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/TwistStamped.h>
 #include <mavros_msgs/Thrust.h>
-
+#include <mav_msgs/RollPitchYawrateThrust.h>
```

Comment:
you need to add `mav_msgs` as a dependency on `CMakeLists.txt` and `package.xml`